Containers/talk/Dockerfile

```diff
@@ -40,33 +38,36 @@ RUN set -ex; \
         lua5.3-dev \
         luarocks5.3; \
     \
+    if [ "$(uname -m )" = "x86_64" ]; then \
+    wget https://eturnal.net/download/linux/eturnal-1.10.1-linux-x64.tar.gz -O - | tar zxC /opt; \
+    elif [ "$(uname -m )" = "aarch64" ]; then \
+    wget https://eturnal.net/download/linux/eturnal-1.10.1-linux-arm64.tar.gz -O - | tar zxC /opt; \
```

Comment:
that needs a workflowthat needs a workflow

Reply:
no, that does not work, as they are compiled for glibc

Reply:
reverted